### PR TITLE
Add Plural's Content Contribution Program

### PR DESCRIPTION
**Pull request by victor-cody**

### CODE REVIEW

**Comment by malgamves** [comment]
hey @victor-cody the webpage you linked says "earn $300" and not $600 as you added - could you edit that and i'll merge

**Comment by victor-cody** [comment]
hey @malgamves, apologies, edited that. I think you're good to go now 💪

File: README.md [hunk 1]
```diff
@@ -187,6 +187,9 @@ A list of companies that have paid Developer Community Writer Programs.
 - [Paperspace](https://blog.paperspace.com/write-for-paperspace/) - $200-$300 per piece
   > Get paid to write articles about machine learning, data science, and more.
 
+- [Plural](https://www.plural.sh/blog/plurals-content-contribution-program/) - $300 per 1,000 word article
+  > Get paid to write articles on, what you have built with Plural, application-specific deployment guides, and architectural comparisons of popular open-     source tools against each other.
```
Comment (malgamves) [inline README.md:191]: there seems to be an extra space here between the dash and "source" - could you look at that? happy to merge after fixing

**Review by malgamves — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (wellsjnr) [reply README.md:191]: I'm interested

Comment (victor-cody) [reply README.md]: hey @malgamves, this is fixed 🙂

**Pull request merged**